web/wp-content/themes/dctx/single.php

```diff
@@ -5,9 +5,21 @@
 
 		<div class="main-content">
 			<?php while (have_posts()) : the_post(); ?>
-				<h1><?php the_title(); ?></h1>
+				<h1 class="post-title"><?php the_title(); ?></h1>
 				<?php dctx_post_meta(); ?>
 
+				<?php if (has_post_thumbnail()): ?>
+					<div class="featured-image">
+						<?php the_post_thumbnail('full'); ?>
+
+						<?php if (get_post( get_post_thumbnail_id())->post_content): ?>
```

Comment:
:no_entry_sign: **Error**: Space between opening control structure and T_COLON is required (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBetweenStructureColon*).